tests/unit/experiments/providers/variant_sets/multi_variant_set_tests.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import collections
+import time
+import unittest
+
+from datetime import timedelta
+
+from baseplate._compat import range
+from baseplate.experiments.providers.variant_sets.multi_variant_set import MultiVariantSet
+
+logger = logging.getLogger(__name__)
+
+
+NUM_BUCKETS_DEF = 1000
+NUM_BUCKETS_ODD = 1037
+
+
+def generate_variant_config():
+    cfg = [
+        {"name": "variant_1", "size": 0.25},
+        {"name": "variant_2", "size": 0.25},
+        {"name": "variant_3", "size": 0.25},
+    ]
+
+    return cfg
+
+def create_multi_variant_set():
+    cfg = generate_variant_config()
+    return MultiVariantSet(cfg)
+
+
+class TestMultiVariantSet(unittest.TestCase):
+
+    def test_validation_passes(self):
+        variant_set = create_multi_variant_set()
+
+        self.assertTrue(isinstance(variant_set, MultiVariantSet))
+
+    def test_validation_fails(self):
+        variant_set_cfg_none = None
+
+        variant_set_cfg_0 = [
+        ]
+
+        variant_set_cfg_2 = [
+            {"name": "variant_1", "size": 0.25},
+            {"name": "variant_2", "size": 0.25},
+        ]
+
+        variant_set_cfg_too_big = [
+            {"name": "variant_1", "size": 0.75},
+            {"name": "variant_2", "size": 0.75},
+            {"name": "variant_3", "size": 0.25},
+        ]
+
+        with self.assertRaises(ValueError):
+            variant_set_none = MultiVariantSet(variant_set_cfg_none)
+        
+        with self.assertRaises(ValueError):
+            variant_set_2 = MultiVariantSet(variant_set_cfg_2)
+        
+        with self.assertRaises(ValueError):
+            variant_set_too_big = MultiVariantSet(variant_set_cfg_too_big)
+
+    def test_distribution_def_buckets(self):
+        variant_set = create_multi_variant_set()
+
+        variant_counts = {
+            "variant_1": 0,
+            "variant_2": 0,
+            "variant_3": 0,
+            None: 0,
+        }
+
+        for bucket in range(0,NUM_BUCKETS_DEF):
```

Comment:
nit: space between `,` and `NUM_BUCKETS_DEF`

Reply:
fixed